cli/azd/pkg/pipeline/github_provider.go

```diff
@@ -627,18 +619,100 @@ func (p *GitHubCiProvider) configurePipeline(
 	provisioningProvider provisioning.Options,
 	additionalSecrets map[string]string,
```

Comment:
Same comment here applies -- Would it be clearer to name it something different than "additional[Secrets|Variables]"? Also, consider reordering such that `ConfigOptions` comes before the secrets/variables since that's the primary argument in my mind...

Reply:
renamed and moved structs to make it easier to follow